README.md

```diff
@@ -1,135 +1,221 @@
-| a customer doesn't sign up, but makes an order just by filling in his email(guest customer) | /customer and then /order to create both resources |
-
-
-<br/><br/>
-## Responses from the endpoints
-The following responses are valid for all of the endpoints.
-
-| Code | Message |  
-| --- | --- |  
-| 204 | Success response. |  
-| 400 | Bad request from client. |  
-| 401 | Project token is invalid. |  
-| 402 | Project requires payment. |  
-| 403 | The IP that the request came from is ignored. |  
-| 500 | Error from server while processing request. |  
-| 502 | Error from server accepting request. |  
+**Note: This guide is intended to be used by developers. If you're not a developer - please find one and come back later :)**
+
+Metrilo helps you track the events that the people visiting your shop do - such as viewing specific pages or products, adding products to the cart, making orders, etc. The provided data is transformed into different reports and analytics metrics you could benefit from.
+
+For more information about Metrilo and its features please visit [the official website](https://www.metrilo.com/).
+
+## Integration overview
+
+:warning: **We strongly recommend doing this on a Staging/Beta environment first to make sure the integration is smooth and nothing breaks down.**
+
+Here is what you need to do in order to integrate your data with Metrilo:
+
+1. Navigate to the `Settings -> Installation` page in your [Metrilo project](https://app.metrilo.com) (if you don't already have one, you can [create one here](https://app.metrilo.com/signup) and note your `API Token` - you'll need it for the next two steps
+2. [Import your resources](#importing-resources) to Metrilo
+3. [Install the tracking library](#tracking-library-installation) on each page of your website that you want to track
+4. Send [tracking events](#sending-tracking-events) to Metrilo for each user action on your website
+
+That's it! Now user actions on your website will be recorded in your [Metrilo project](https://app.metrilo.com).
+
+## API
+
+Our API documentation is written according to the [OpenAPI Specification](https://swagger.io/docs/specification/about/). You can find all of the endpoints the Metrilo API serves and what parameters they require [here](https://app.swaggerhub.com/apis/metrilo/api/1.0.0).
+
+All calls to these endpoints have to be done from your backend - therefore, we don't provide any specific code examples as the implementation is bound to your backend logic and programming language.
+
+:warning: Each Metrilo project has its own `API Token` and multiple stores cannot point to the same Metrilo project.
+
+### Importing resources
+
+A _resource_ is a **category**, **product**, **customer** or **order**. Any time one of these changes on your backend, you should call the respective endpoint to create or update the resource in Metrilo.
```

Comment:
as I wrote above, I think 'Any time one of these changes on your backend...' should be a step on its own

Reply:
👍 Will add a `Updating resources` section after the `Import` one

Reply:
actually, it have to be new and updated resources (new orders, new products)

Reply:
`Creating and updating resources`? :)

Reply:
sounds good